libs/hdf-converters/src/xccdf-results-mapper.ts

```diff
@@ -481,37 +481,41 @@ export class XCCDFResultsMapper extends BaseConverter {
             },
             descriptions: [
               {
-                data: {
-                  path: ['check.check-content-ref.name'],
-                  transformer: (text: string | string[]) =>
-                    asArray(text).join('\n') || undefined
-                },
-                label: 'check'
-              },
+                path: ['check.check-content-ref.name'],
+                transformer: (
+                  data: string | string[]
+                ): ExecJSON.ControlDescription => ({
+                  data: asArray(data).join('\n'),
+                  label: 'check'
+                })
+              } as any,
```

Comment:
would prefer to avoid using 'any' if possible?  how does it complain if you don't cast it?

if we can't make it happy by clearing up the types directly, then i usually prefer doing 'as unknown as whatever_it_ought_to_be' over 'as any'

Reply:
The type of `descriptions` is `ControlDescription` and that has two mandatory fields: `data` and `label`. Neither of those fields are directly specified (they are set by the transformer, which typescript has no understanding of).

I implemented the `as unknown as ControlDescription` approach.